docs/src/advanced/dynamic_tiler.md

```diff
@@ -80,7 +80,7 @@ def tilejson(
     url: str = Query(..., description="Cloud Optimized GeoTIFF URL."),
 ):
     """Return TileJSON document for a COG."""
-    tile_url = request.url_for("tile", {"z": "{z}", "x": "{x}", "y": "{y}"})
+    tile_url = request.url_for("tile", z='{z}', x='{x}', y='{y}')
```

Comment:
```suggestion
    tile_url = str(request.url_for("tile", z="{z}", x="{x}", y="{y}"))
```

This is to avoid issue we latest starlette/fastapi versions